src/app/overview/overview.component.html

```diff
@@ -1,34 +1,117 @@
 <div class="row d-flex justify-content-center">
-    <h2 class="m-5">Architecture</h2><br>
+    <h2 class="m-5">Why Goblint?</h2><br>
 </div>
 <div class="row d-flex justify-content-center mb-5">
-    <div class="col"><p> 
-        Goblint is implemented in OCaml and uses an updated fork of CIL as its parser frontend for the C language.
-        The architecture of Goblint is designed to be modular. 
-        Analyses, which are defined by their abstract domains and transfer functions, can be activated via runtime configuration options. 
-        A flexible query system allows for communication between analyses. 
-        Together, the combined analyses and the control-flow graphs of the functions in the program provide the side-effecting constraint system, 
-        which is solved by some local generic solver. 
-        Post-processing the solution yields results for the analysis.
+    <div class="col"><p>
+        Developing software that is free of bugs is a complex and challenging task, and even more so in a multi-threaded setting.
+        Goblint supports developers in this endeavor, by providing insights into the code and <b>flagging potential bugs</b>.
+        This flagging of bugs goes way beyond what inters typically can provide: Instead Goblint carefully considers the semantics
+        of the entire program to flag all the bugs that could occur.
```

Comment:
"Flagging bugs" makes us sound like yet another bug finder. I think this should emphasize soundness.

Reply:
I now made `all` bold, is this the direction you suggested we should go here?